src/main/java/org/folio/service/mapping/RuleFactory.java

```diff
@@ -88,16 +89,18 @@
     return rules;
   }
 
-  private void markItemTransformationsWithSameFieldInHoldings(List<Transformations> mappingTransformations) {
-    for (Transformations holdingsTransformation: mappingTransformations) {
-      for (Transformations itemTransformation: mappingTransformations) {
-        if (holdingsTransformation.getRecordType().equals(HOLDINGS) && itemTransformation.getRecordType().equals(ITEM)) {
```

Comment:
If this logic is removed how will we identify the scenario where holdings and items are given under the same tag, which was handled in MDEXP-266?